11 - Custom Video Player/scripts.js

```diff
@@ -0,0 +1,103 @@
+class CustomVideoPlayer {
+    constructor(selector) {
+        this.player = document.querySelector(selector);
+        this.video = this.player.querySelector('.viewer');
+        this.progress = this.player.querySelector('.progress');
+        this.progressBar = this.player.querySelector('.progress__filled');
+        this.toggle = this.player.querySelector('.toggle');
+        this.skipButtons = this.player.querySelectorAll('[data-skip]');
```

Comment:
odd that all selectors are done via classes but this one via data. did wes say why he chose it like this?

Reply:
Wes mentions that he uses it in order to select any buttons with `data-set`, in case we want to add more buttons in the future. Nothing more.